content/en/docs/aws/customizing-aws.md

```diff
@@ -87,4 +87,4 @@ Please see [this section](/docs/aws/iam-for-sa)
 Please see [this section](/docs/aws/private-access)
```

Comment:
```suggestion

Check the [How to create private EKS clusters](/docs/aws/private-access) guide.
```

("see" -> "refer to" or "go to" for accessibility)
(use full name of guides for better UX)